src/actions/engagementActions.js

```diff
@@ -9,14 +9,14 @@ export const DISENGAGE_SUCCESS = 'DISENGAGE_SUCCESS'
 export const DISENGAGE_FAILURE = 'DISENGAGE_FAILURE'
 export const ENGAGEMENTS = 'ENGAGEMENTS'
 
-export const engage = (incidentId, participant, timeEngaged = moment()) =>
+export const engage = (incidentId, participant, timeEngaged = DateTime.local()) =>
```

Comment:
Engagements should be tracked only in UTC

Reply:
And yes, I realize that the existing code probably didn't track in UTC. Sorry about that.